### PR TITLE
refactor: use async function and support egg@2

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: app/middleware/cors.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('kcors');
+module.exports = require('@koa/cors');
```
Comment (dead-horse) [inline app/middleware/cors.js:3]: kcors renamed to @koa/cors now

Comment (atian25) [reply app/middleware/cors.js:3]: will others koa official lib rename?

**Review by atian25 — APPROVED** [review]

**Review by fengmk2 — APPROVED** [review]
LGTM

File: app/middleware/cors.js (hunk 1 above)
Comment (dead-horse) [reply app/middleware/cors.js:3]: I don't know, just because kcors is to strange

**Comment by atian25** [comment]
@dead-horse 发

**Pull request merged**